recipe/meta.yaml

```diff
@@ -23,8 +22,8 @@ requirements:
 
 outputs:
   - name: pre-commit
-    script: build_base.bat  # [win]
-    script: build_base.sh  # [not win]
+    build:
+      script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
I'm not sold on this. I'd prefer if we find the real problem then doing this.

Reply:
My suggestion is that the issue is caused by `pip` or `virtualenv` (or incompatible versions) but I haven't any proves yet

Reply:
cf pre-commit is a noarch package. It doesn't include file pre-commit.exe, unlike our package.

Reply:
`pre_commit` is noarch but `pre-commit` has a per python build.

Reply:
Ah wait, by `cf` you mean conda-forge?

Reply:
yes, if you extract both the difference is there

Reply:
If I make a noarch package then the error will be the same